Files origsrc/pythonnet-1.0-rc2-py2.4-clr1.1-src/CLR.dll and src/pythonnet-1.0-rc2-py2.4-clr1.1-src/CLR.dll differ
--- origsrc/pythonnet-1.0-rc2-py2.4-clr1.1-src/Python.Runtime.dll.config	1969-12-31 18:00:00.000000000 -0600
+++ src/pythonnet-1.0-rc2-py2.4-clr1.1-src/Python.Runtime.dll.config	2007-03-08 13:18:50.515312200 -0600
@@ -0,0 +1,3 @@
+<configuration>
+    <dllmap dll="python24" target="libpython2.4.dll"/>
+</configuration>
--- origsrc/pythonnet-1.0-rc2-py2.4-clr1.1-src/makefile	2005-06-05 10:56:11.000000000 -0500
+++ src/pythonnet-1.0-rc2-py2.4-clr1.1-src/makefile	2007-03-08 13:26:42.453559000 -0600
@@ -2,10 +2,10 @@
 # Camilo Uribe <kmilo@softhome.net> for contributing Mono support.
 
 RELEASE=pythonnet-1.0-rc2-py2.4-clr1.1-src
-RUNNER=
-ILDASM=ildasm
+RUNNER=mono
+ILDASM=monodis
 ILASM=ilasm
-CSC=csc.exe
+CSC=mcs
 
 all: python.exe CLR.dll Python.Test.dll
 
@@ -22,7 +22,7 @@
 	$(CSC) /nologo /unsafe /target:library /out:../../Python.Runtime.dll \
         /recurse:*.cs
 	cd ..; cd ..;
-	$(ILDASM) /nobar Python.Runtime.dll /out=Python.Runtime.il;
+	$(ILDASM) --output=Python.Runtime.il Python.Runtime.dll;
 	$(RUNNER) ./callconvutil.exe;
 	$(ILASM) /nologo /quiet /dll \
 	/resource=Python.Runtime.res /output=Python.Runtime.dll \
@@ -42,6 +42,7 @@
 	cd src; cd testing; \
 	$(CSC) /nologo /target:library /out:../../Python.Test.dll \
 	/reference:../../Python.Runtime.dll \
+	/reference:System.Windows.Forms \
 	/recurse:*.cs
 	cd ..; cd ..;
 
@@ -87,7 +88,7 @@
 
 
 dis:
-	$(ILDASM) Python.Runtime.dll /out=Python.Runtime.il
+	$(ILDASM) --output=Python.Runtime.il Python.Runtime.dll
 
 
 asm:
